plugins/web/opentelemetry-plugin-react-load/test/test-react-components/AllLifecycles.tsx

```diff
@@ -9,6 +9,18 @@ export default class AllLifecycles extends React.Component {
         console.log("mounted");
     }
 
+    componentDidUpdate(prevProps: any){
+        console.log("updated");
```

Comment:
This appears to be a debugger line. I'm not sure where in the lifecycle you feel you are at the moment. But this should be removed in a final version.